### ISSUE TITLE
[Feature Request] - implement cmd::cancel for select

### ISSUE THREAD

**Issue by tramhao** [veeso/tui-realm-stdlib#6]
## Description

When in select, I want to escape the change.

### Changes

My code is as below:
```rust
            Event::Keyboard(KeyEvent {
                code: Key::Esc | Key::Char('q'),
                ..
            }) => match self.state() {
                State::One(_) => return Some(Msg::KeyEditor(KEMsg::KeyEditorCloseCancel)),
                _ => self.perform(Cmd::Cancel),
            },
```
Currently this code is not working.

**Comment by veeso** [comment]
Sorry, I didn't get what it is supposed to do? Should it clear the current selection?

P.s. for a period I'm not going to make any commit to my projects, unless in case of critical issues, so for the moment remember that you can always access the states with `self.component.states.*`

**Comment by tramhao** [comment]
Recently I'm using select a lot. There are several issues:
1. When press enter, you'll have to press enter again to confirm the select. If you press cancel, there is not function to return to original selection. 
2. After press enter, if you press tab, the select will remain open and no action confirmed. I think in this scenario it's best to cancel it as well.
Let me think about the implementation on client side. Thanks.

**Comment by veeso** [comment]
Hi,

sorry for the late answer, but I've taken a break for the last month.

I've implemented your requests in the 1.1.6 version.
